### PR TITLE
gettext 

### PR DESCRIPTION
**Pull request by scopatz**
Forgot that I needed this before glibmm would work.

### CODE REVIEW

File: recipes/gettext/build.sh [hunk 1]
```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
+set -x
+set -e
+
+./configure --prefix=$PREFIX
+make
+make install
+make check
```
Comment (ocefpaf) [inline recipes/gettext/build.sh]: This should be in the line above, no?

File: recipes/gettext/build.sh [hunk 2]
```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
+set -x
+set -e
```
Comment (ocefpaf) [inline recipes/gettext/build.sh]: Not sure if those will be useful here.

File: recipes/gettext/build.sh (hunk 1 above)
Comment (scopatz) [inline recipes/gettext/build.sh]: Yep Sorry,

File: recipes/gettext/build.sh (hunk 2 above)
Comment (scopatz) [inline recipes/gettext/build.sh]: Thanks, I got rid of them.

File: recipes/gettext/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+  url: ftp://ftp.gnu.org/pub/gnu/gettext/gettext-{{ version }}.tar.xz
+  sha256: 378fa86a091cec3acdece3c961bb8d8c0689906287809a8daa79dc0c6398d934
+
+build:
+  number: 0
+  skip: true  # [win]
+
+requirements:
+  build:
+    - libffi
```
Comment (ocefpaf) [inline recipes/gettext/meta.yaml]: @stefan-balke had trouble with `libffi` on OS X.

https://github.com/conda-forge/staged-recipes/pull/234#issuecomment-212321103

Comment (scopatz) [inline recipes/gettext/meta.yaml]: Thanks for letting me know!

**Comment by scopatz** [comment]
looks like this builds ok

**Comment by pelson** [comment]
> looks like this builds ok

:+1:

**Pull request merged**